src/hotspot/share/runtime/lockStack.hpp

```diff
@@ -30,8 +30,9 @@
 #include "utilities/globalDefinitions.hpp"
 #include "utilities/sizes.hpp"
 
-class Thread;
+class JavaThread;
```

Comment:
Seems an unrelated change.

Reply:
This is a minor cleanup of the prototypes. lockStack.hpp uses `JavaThread*`, not `Thread*`. Do you prefer not to touch it in this PR?

Reply:
If it uses `JavaThread` but only had `Thread` then it must be including it some other way, so the prototype seems unnecessary. But okay to leave it.